server/public/plugin/api.go

```diff
@@ -1207,6 +1207,73 @@ type API interface {
 	// @tag User
 	// Minimum server version: 9.3
 	UpdateUserAuth(userID string, userAuth *model.UserAuth) (*model.UserAuth, *model.AppError)
+
+	// RegisterPluginForSharedChannels registers the plugin as a `Remote` for SharedChannels.
+	// The plugin will receive synchronization messages via the `OnSharedChannelsSyncMsg` hook.
+	// This API is idempotent - when called repeatedly with the same `RegisterPluginOpts.PluginID`
+	// it will return the same remoteID.
+	//
+	// @tag SharedChannels
+	// Minimum server version: 9.4
```

Comment:
I think this is not going to be merged on time for 9.4 I think we should update all to 9.5

Reply:
Fixed for plugin APIs and hooks.